sklearn/neighbors/kde.py

```diff
@@ -71,12 +75,13 @@ class KernelDensity(BaseEstimator):
         :class:`BallTree` or :class:`KDTree`.
     """
     def __init__(self, bandwidth=1.0, algorithm='auto',
-                 kernel='gaussian', metric="euclidean", atol=0, rtol=0,
+                 kernel='gaussian', metric="minkowski", p=2, atol=0, rtol=0,
```

Comment:
As with dbscan

Reply:
@jnothman 
How do I get the error that occurred in ci/circleci: python2? Can you help me?

Reply:
@gilbertoolimpio  You can ignore the circleci/python2 logs, since this PR will go in v0.21 if goes through, which doesn't support python2 anyway.

Reply:
@adrinjalali 

Ok! Thank you for this explanation!